recipes/src/recipes/RecipeDetail.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import styles from '../styles'
+import findRecipe from './findRecipe'
+
+const {
+  RecipeDetailContainer,
+  RecipeDetailName,
+  RecipeDetailDescription,
+  IngredientListItem,
+  IngredientListBox,
+  IngredientListContainer,
+  RecipeDetailButtons,
+  RecipeDetailButton,
+} = styles;
+
+
+function IngredientList(props) {
+  if(!props.ingredients) {
+    return null;
+  }
+  const list = props.ingredients.map((ingredient) =>
+    <IngredientListItem key={ingredient.id}>{ingredient.name}</IngredientListItem>
+  )
+  return <IngredientListBox>
+    <IngredientListContainer>
+      {list}
+    </IngredientListContainer>
+  </IngredientListBox>
+};
```

Comment:
This looks like a component, maybe it should be in a separate file?

Reply:
Yeah - I was being lazy!
😉 
I'll pull it out.
(And add some tests)